packages/dso-toolkit/src/components/scroll-container/scroll-container.args.ts

```diff
@@ -0,0 +1,21 @@
+import { HandlerFunction } from "@storybook/addon-actions";
+import { ArgTypes } from "../../storybook/index.js";
+
+import { ScrollContainer } from "./scroll-container.models.js";
+
+export interface ScrollContainerArgs {
+  dsoScrollContainerEvent: HandlerFunction;
+}
+
+export const scrollContainerArgTypes: ArgTypes<ScrollContainerArgs> = {
+  dsoScrollContainerEvent: {
+    action: "scrollEnd",
```

Comment:
Er is geen vaste lijn, maar wat mij betreft is in dit geval duidelijker om de volledige event name te tonen in Storybook: `dsoScrollContainerEvent`.

Reply:
Eens